relay/sources/rudder-server-relay-postinst

```diff
@@ -76,13 +83,6 @@ if [ ! -f /var/rudder/lib/ssl/nodescerts.pem ]; then
   touch /var/rudder/lib/ssl/nodescerts.pem
```

Comment:
shoudl be allnodescerts.pem

Reply:
(only needed and generated on root server, but does no harm if present elsewhere)

Reply:
Done: it will be empty anyway